catalog/files/api/src/main/java/org/projectnessie/catalog/files/config/BucketOptions.java

```diff
@@ -15,4 +15,10 @@
  */
 package org.projectnessie.catalog.files.config;
 
-public interface BucketOptions {}
+import java.util.Map;
+
+public interface BucketOptions {
+
+  /** Iceberg table configuration overrides for all tables stored in this bucket. */
+  Map<String, String> tableConfigOverrides();
```

Comment:
Can you mention this new property in the CHANGELOG.md?

Reply:
At least the ones for for the default ones - i suspect not many users have bucket-specific configs.